src/main/resources/graphql/service/quiz.graphqls

```diff
@@ -134,10 +134,226 @@ type MultipleChoiceAnswer {
     feedback: String
 }
 
+type ClozeQuestion implements Question {
+    """
+    The elements of the cloze question.
+    """
+    clozeElements: [ClozeElement!]!
+    """
+    All selectable answers for the blanks (computed). This contains the correct answers as well as wrong answers.
+    """
+    allBlanks: [String!]!
+    """
+    Whether the blanks must be answered in free text or by selecting the correct answer from a list.
+    """
+    showBlanksList: Boolean!
+
+    # inherited from Question
+    """
+    Unique identifier of the question.
+    """
+    id: UUID!
+    """
+    Number of the question, i.e., the position of the question in the list of questions.
+    Only relevant if questionPoolingMode is ORDERED.
+    """
+    number: Int!
+    """
+    Type of the question.
+    """
+    type: QuestionType!
+    """
+    Optional hint for the question, can be markdown.
+    """
+    hint: String
+}
+
+
+interface ClozeElement {
+    """
+    Type of the element.
+    """
+    type: ClozeElementType!
+}
+
+type ClozeTextElement implements ClozeElement {
+    """
+    Text of the element, can be markdown.
+    """
+    text: String!
+
+    # inherited from ClozeElement
+    """
+    Type of the element. Must be TEXT.
+    """
+    type: ClozeElementType!
+}
+
+type ClozeBlankElement implements ClozeElement {
+    """
+    The correct answer for the blank.
+    """
+    correctAnswer: String!
+    """
+    Wrong answers for the blank.
+    """
+    wrongAnswers: [String!]!
+    """
+    Feedback for the blank when the user selects a wrong answer, can be markdown.
+    """
+    feedback: String
+
+    # inherited from ClozeElement
+    """
+    Type of the element. Must be BLANK.
+    """
+    type: ClozeElementType!
+}
+
+enum ClozeElementType {
+    TEXT
+    BLANK
+}
+
+"""
+Association question, i.e., a question where the user has to assign the correct right side to each left side.
+"""
+type AssociationQuestion implements Question {
```

Comment:
question where you have to assign the correct element on the right side to the correct element on the left